staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/customresource_discovery_controller.go

```diff
@@ -113,6 +116,11 @@ func (c *DiscoveryController) sync(version schema.GroupVersion) error {
 			if v.Name == version.Version {
 				foundThisVersion = true
 			}
+			if v.Storage {
+				bytes := sha256.Sum256([]byte(gv.String()))
```

Comment:
One last random thought-- since we really don't need 256 bits of entropy here, maybe switch to Sum224?

I guess https://golang.org/pkg/crypto/md5/ is even shorter.

Maybe best is to just keep the code here and add a `[:16]` at the end.

It just seems like having the strings of gibberish be as short as possible will be slightly nicer to humans.

This is optional, but think about it since it'll be hard to change in the future...